packages/p2p/stores/my-profile-store.js

```diff
@@ -59,9 +59,9 @@ export default class MyProfileStore extends BaseStore {
     handleSubmit(values) {
         requestWS({
             p2p_advertiser_update: 1,
-            contact_info: values.contact_info,
-            payment_info: values.payment_info,
-            default_advert_description: values.default_advert_description,
+            contact_info: values.contact_info.replace(/^\s+|\s+$|\s+(?=\s)/g, ''),
+            payment_info: values.payment_info.replace(/^\s+|\s+$|\s+(?=\s)/g, ''),
+            default_advert_description: values.default_advert_description.replace(/^\s+|\s+$|\s+(?=\s)/g, ''),
```

Comment:
How about `trim()`? 🤔 

```suggestion
            contact_info: values.contact_info.trim(),
            payment_info: values.payment_info.trim(),
            default_advert_description: values.default_advert_description.trim(),
```

Reply:
Trim is used to remove whitespace characters from the start and end of a string whereas this regex removes extra whitespace in between a string of words as well.